tests/contrib/openai/utils.py

```diff
@@ -36,6 +36,7 @@ def _expected_llmobs_tags(error=None):
         "env:",
         "service:",
         "source:integration",
+        "ml_app:",
```

Comment:
Should this have a default value? In the tracer we we set service names to  "unnamed_python_service"

Reply:
Thanks for the catch!